src/pool/srv_pool.c

```diff
@@ -5210,6 +5256,32 @@ ds_pool_update_handler(crt_rpc_t *rpc)
 	pool_target_addr_list_free(&inval_list_out);
 }
 
+static int
+pool_svc_reint_rank(struct pool_svc *svc, d_rank_t rank)
```

Comment:
this is dup of pool_svc_exclude_rank(), can we just change it to a generic function, then define two wrapper functions for it

```
pool_svc_change_rank(....int opt)
{
         ....
         rc = pool_svc_update_map(svc, opt, ....);
}

int
pool_svc_reint_rank(...)
{
       return pool_svc_change_rank(svc, rank, POOL_REINT);
}
```